### ISSUE TITLE
fail to parse with no error

### ISSUE THREAD

**Issue by christiancadieux** [slok/go-helm-template#63]
this chart does not return any template

```
apiVersion: batch/v1
kind: Job
metadata:
  name: eg-{{.Values.cfg.tenant}}-gateway-helm-certgen
  namespace: {{ .Values.cfg.controllerNamespace }}
  labels:
    helm.sh/chart: gateway-helm-latest
    app.kubernetes.io/name: gateway-helm
    app.kubernetes.io/instance: eg-{{.Values.cfg.tenant}}
    app.kubernetes.io/version: "latest"
    app.kubernetes.io/managed-by: Helm
  annotations:
    "helm.sh/hook": pre-install, pre-upgrade
spec:
  backoffLimit: 1
  completions: 1
  parallelism: 1
  template:
    metadata:
      labels:
        app: certgen
    spec:
      containers:
      - command:
        - envoy-gateway
        - certgen
        env:
        - name: ENVOY_GATEWAY_NAMESPACE
          valueFrom:
            fieldRef:
              apiVersion: v1
              fieldPath: metadata.namespace
        - name: KUBERNETES_CLUSTER_DOMAIN
          value: cluster.local
        image: {{ .Values.eg.certgen.image }}
        imagePullPolicy: IfNotPresent
        name: envoy-gateway-certgen
      imagePullSecrets: []
      restartPolicy: Never
      securityContext:
        runAsGroup: 65534
        runAsNonRoot: true
        runAsUser: 65534
      serviceAccountName: eg-{{.Values.cfg.tenant}}-gateway-helm-certgen
  ttlSecondsAfterFinished: 30
```
value.yaml

```
eg:
  envoyproxy:
    replicas: 2
    image: hub.comcast.net/k8s-eng/envoyproxy/envoy:distroless-v1.29.3
    resources:
      limits:
        cpu: 1000m
        memory: 1024Mi
      requests:
        cpu: 256m
        memory: 256Mi

  shutdownManager:
    image: hub.comcast.net/k8s-eng/envoyproxy/gateway:v1.1.0
  rateLimit:
    image: hub.comcast.net/k8s-eng/envoyproxy/ratelimit:master
  certgen:
    image: hub.comcast.net/k8s-eng/envoyproxy/gateway:v1.1.0
  gateway:
    replicas: 1
    image: hub.comcast.net/k8s-eng/envoyproxy/gateway:v1.1.0
    resources:
      limits:
        cpu: 500m
        memory: 1024Mi
      requests:
        cpu: 100m
        memory: 256Mi
  testDeployment:
    image: hub.comcast.net/k8s-eng/gcr.io/k8s-staging-gateway-api/echo-basic
    resources:
      limits:
        cpu: 200m
        memory: 1024Mi
      requests:
        cpu: 100m
        memory: 256Mi
cfg:
  tenant: "tenant2"
  tsfService: "service1"
  rail: "public"
  lbIPs: "10.112.182.52"
  controllerNamespace: tenant2-ns1
  namespaces:
    - tenant2-ns1
    - tenant2-ns2
```

**Comment by slok** [comment]
Hi @christiancadieux!

This library doesn't support hooks, only release manifest templates. Your chart only has a hook, so that's not part of the chart release and thats why it returns "empty".

I you would want to be part of the final manifest release, remove the `"helm.sh/hook": pre-install, pre-upgrade` annotation.

**Comment by slok** [comment]
I released [v0.7.0](https://github.com/slok/go-helm-template/releases/tag/v0.7.0) with the option to render hooks. In case you think would be better than removing the annotation for your use case.

**Comment by christiancadieux** [comment]
great - thanks
